forms-html/Ubaida-Vazhin/index.html

```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+
+
+<form action="action_page.php" method="POST">
+    <br>
+    <label for="email">enter your email</label><br>
+    <input type="text" placeholder="Enter Email" name="email" id="email" required><br><br>
```

Comment:
please make sure that you are using the correct input type here, also the placeholder attribute specifies a short hint that describes the expected value of an input field, so it's better to be like
placeholder="example@example.com"